mno_analysis_tools/run-mno-analysis.sh

```diff
@@ -38,5 +52,10 @@ echo "Starting run with id '$RUN_ID'"
 ./docker-run-compute-messages-per-period.sh --profile-memory ./data "$RAW_MESSAGES_FILE_PATH" "$TARGET_OPERATOR" \
     "$TARGET_MESSAGE_DIRECTION" "$START_DATE" "$END_DATE" "$TIME_FRAME" "$OUTPUT_DIR"
 
-./docker-run-compute-msg-difference-btwn-two-firebase-time-periods.sh --profile-memory ./data "$RAW_MESSAGES_FILE_PATH" \
+if [[ "$OPTIONAL_TIME_FRAME" = true ]]; then
+    ./docker-run-compute-msg-difference-btwn-two-firebase-time-periods.sh --profile-memory ./data "$RAW_MESSAGES_FILE_PATH" \
+    "$TARGET_OPERATOR" "$TARGET_MESSAGE_DIRECTION" "$START_DATE" "$END_DATE" "$OUTPUT_DIR" "$OPTIONAL_TIME_FRAME_VALUE"
```

Comment:
If this is optional move it to an optional flag instead. See https://github.com/AfricasVoices/Project-WorldVision/blob/master/run_scripts/5_automated_analysis.sh for an example of how to accept an optional arg to this script and forward it into another shell script more cleanly, without needing an if.

Reply:
Thanks